tailwind.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   darkMode: "class",
   theme: {
     extend: {
+      screens: { 'mobile': { 'max': '640px' } },
```

Comment:
I don't think we need to add new screen sizes

Reply:
sm is ->@media (min-width: 640px)
and the mobile -> @media (max-width: 640px)
if you try to use sm all screens less than 640px on mobile won't be affected and mobile screen resolution is usually  between 412 and 414 pixels wide 
and this style I'm applying must only be visible in mobile view so a new breakpoint in my opinion is the better option for both future responsive edits as this project wasn't done in a mobile-first approach  and also not to make any bugs in already existed code

Reply:
But we can do the reverse (which is what tailwind recommends), the default is for mobile, then use `sm-` for anything larger - this results in `max-width: 640px` for anything smaller

> [Working mobile-first](https://tailwindcss.com/docs/responsive-design#working-mobile-first)
> By default, Tailwind uses a mobile-first breakpoint system, similar to what you might be used to in other frameworks like Bootstrap.
> What this means is that unprefixed utilities (like uppercase) take effect on all screen sizes, while prefixed utilities (like md:uppercase) only take effect at the specified breakpoint and above.

> Where this approach surprises people most often is that to style something for mobile, you need to use the unprefixed version of a utility, not the sm: prefixed version. Don’t think of sm: as meaning “on small screens”, think of it as “at the small breakpoint“.

https://tailwindcss.com/docs/responsive-design#working-mobile-first